notebooks/2018-11-27-Value-when-other-is-max.ipynb

```diff
@@ -371,7 +371,7 @@
    "metadata": {},
    "source": [
     "As you see, `numpy` is unable to broadcast the different shapes together. `xarray` is much better at broadcasting as it uses all the metadata stored in the DataArray to identify the dimensions in each array.\n",
```

Comment:
The reader might not know what broadcasting is about. I only knew because you wrote about it in your previous version. :)
Maybe say: "The reason why this failed is that, in order for `where()` to work, it first has to make the shapes of the structures match. It does this by broadcasting the smaller structure (in this case the 1d `tas.time` array) onto the larger structure (the 3d `tas` array), treating the `tas.time` array as a 3d array, with equal time values across all spatial points. As you can see, `numpy` is ..."